### PR TITLE
Update to Angular v4.1.3

### LINKED ISSUE #1141: Update angular version in Augury frontend

### PR DESCRIPTION
**Pull request by stevenkampen**
resolves #1141

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -35,14 +35,14 @@
...
     "@angular-redux/store": "^6.4.1",
-    "@angular/common": "2.4.5",
-    "@angular/compiler": "2.4.5",
-    "@angular/core": "2.4.5",
-    "@angular/forms": "2.4.5",
-    "@angular/http": "2.4.5",
-    "@angular/platform-browser": "2.4.5",
-    "@angular/platform-browser-dynamic": "2.4.5",
-    "@angular/router": "3.0.0",
+    "@angular/common": "4.1.3",
```
Comment (igor-ka) [inline package.json]: @stevenkampen let me know if it makes sense to add ^ or not

Comment (stevenkampen) [inline package.json]: @igor-ka I would rather lock everything down, just because I don't see any benefit to the opposite. Any opinion?

Comment (JinSY) [inline package.json]: @stevenkampen I'm in agreement. I would rather keep it locked down

**Comment by stevenkampen** [comment]
@igor-ka Please review again. Had to fix some compilation issues after updating yarn.lock.

**Pull request merged**